controller/scripts.go

```diff
@@ -18,6 +18,7 @@
 
 package controller
 
+/* TODO Delete this file
```

Comment:
I think it's better to keep it and keep the execute transaction command on it.

Reply:
I think it's more simple to just have the file controller that checks the file type and executes as a script, transaction, or contract based on that. Can change if this really bothers you though.